lms/envs/common.py

```diff
@@ -1860,6 +1860,9 @@
     # Microsite configuration
     'microsite_configuration',
 
+    # RSS Proxy
+    'rss_proxy',
```

Comment:
I think it would be better to import this new app (all new djangoapps, actually) in a manner similar to edx-milestones and edx-proctoring -- separate repo + package, specify dependency via requirements file, import via OPTIONAL_APPS.